app/pktgen-cmds.c

```diff
@@ -279,6 +304,39 @@ pktgen_script_save(char *path)
         fprintf(fd, "range %d dst port max %d\n", i, range->dst_port_max);
         fprintf(fd, "range %d dst port inc %d\n", i, range->dst_port_inc);
 
+        fprintf(fd, "\n");
+        fprintf(fd, "range %d tcp flag clr all\n", i);
+        if (range->tcp_flags & URG_FLAG) {
+            fprintf(fd, "range %d tcp flag set %s\n", i, "urg");
+        }
```

Comment:
For coding style I try not to have single line 'if', 'for', 'while', ... statements with {} around. Can you please remove the {} around these single lines.